src/index.ts

```diff
@@ -1,167 +1,9 @@
 #!/usr/bin/env node
 
 /* tslint:disable:no-console */
-import * as Pino from 'pino'
-import 'reflect-metadata'
+/* tslint:disable:no-relative-imports */
+import { app } from './app'
```

Comment:
You should be able to avoid this `/* tslint:disable:no-relative-imports */` by changing the import to just `app` (without the `./`). Can you give that a try?

Reply:
done.